deployment/terraform/output.go

```diff
@@ -48,17 +51,18 @@ type output struct {
 // Output contains the output variables which are
 // created after a deployment.
 type Output struct {
-	ClusterName         string
-	Proxy               Instance            `json:"proxy"`
-	Instances           []Instance          `json:"instances"`
-	DBCluster           DBCluster           `json:"dbCluster"`
-	Agents              []Instance          `json:"agents"`
-	MetricsServer       Instance            `json:"metricsServer"`
-	ElasticSearchServer ElasticSearchDomain `json:"elasticServer"`
-	JobServers          []Instance          `json:"jobServers"`
-	S3Bucket            S3Bucket            `json:"s3Bucket"`
-	S3Key               IAMAccess           `json:"s3Key"`
-	DBSecurityGroup     []SecurityGroup     `json:"dbSecurityGroup"`
+	ClusterName          string
+	Proxy                Instance            `json:"proxy"`
+	Instances            []Instance          `json:"instances"`
+	DBCluster            DBCluster           `json:"dbCluster"`
+	Agents               []Instance          `json:"agents"`
+	MetricsServer        Instance            `json:"metricsServer"`
+	ElasticSearchServer  ElasticSearchDomain `json:"elasticServer"`
```

Comment:
Let's also display the domain name after deployment create/info?

Reply:
Do you mean the endpoint? It's already there: https://github.com/mattermost/mattermost-load-test-ng/blob/master/deployment/terraform/info.go#L68

Reply:
Oh looks like they were already added.